exercises/binary/test/test_binary.c

```diff
@@ -0,0 +1,85 @@
+#include "vendor/unity.h"
+#include "../src/binary.h"
+
+void b1_is_decimal_1(void)
```

Comment:
I'd imagine @palisaide meant `void test_b1_is_decimal_1(void)`, i.e. prefix test function names with `test_`, as this matches the convention used with the test framework in use and all other test files in xc.